modes.xml

```diff
@@ -61,12 +61,16 @@
       <program name="apertium-tagger -g $2">
         <file name="pt-es.prob"/>
       </program>
+      <program name="apertium-restore-caps">
+        <file name="pt-es.crx.bin"/>
+      </program>
       <program name="apertium-pretransfer"/>
       <program name="apertium-transfer">
         <file name="pt-es.t1x"/>
         <file name="pt-es.t1x.bin"/>
         <file name="pt-es.autobil.bin"/>
       </program>
+      <program name="apertium-extract-caps"/>
```

Comment:
you have these backwards - extracting is right after morphological analysis and restoring is right after generation